frontend/app/src/locales/gr.json

```
@@ -2035,13 +2035,19 @@
     "tx_url": "Transactions will open at {txUrl}"
   },
   "external_services": {
+    "api_key": "API key",
     "beaconchain": {
       "delete_tooltip": "Deletes the Beaconcha.in API key",
       "description": "rotki uses Beaconcha.in to obtain information about ETH staking in your portfolio. An API key is only needed if you are being rate-limited.",
       "hint": "Enter your Beaconcha.in API key",
-      "label": "API key",
       "title": "Beaconcha.in"
     },
+    "blockscout": {
+      "delete_tooltip": "Διαγράφει το κλειδί Blockscout API",
+      "description": "rotki χρησιμοποιεί το Blockscout για να λάβει πληροφορίες σχετικά με ερωτήματα ανάληψης ETH",
```

```suggestion
      "description": "Το rotki χρησιμοποιεί το Blockscout για να λάβει πληροφορίες σχετικά με τα ερωτήματα ανάληψης ETH",
```

deepl's translation is always way more accurate, I only had to change a word.